src/pages/landingPage.component.ts

```diff
@@ -11,8 +11,10 @@ export class LandingPageComponent implements OnInit {
   customerLoginPressed: boolean = false;
   vendorLoginPressed: boolean = false;
   username: string;
```

Comment:
Username and password should be put into one object that you can reference from the HTML. Something like:
```
loginForm: {
  username: string,
  password: string,
} = {
  username: '',
  password: '',
};